examples/lda.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import absolute_import, division, print_function
+
+import argparse
+import logging
+
+import torch
+from torch import nn
+from torch.distributions import constraints
+
+import pyro
+import pyro.distributions as dist
+from pyro import poutine
+from pyro.infer import SVI, JitTraceEnum_ELBO, TraceEnum_ELBO
+from pyro.optim import Adam
+
+logging.basicConfig(format='%(relativeCreated) 9d %(message)s', level=logging.INFO)
+
+
+# This is a fully generative model of a batch of documents.
```

Comment:
itd be useful to link the paper in the description or if you want to avoid getting into topic model details, maybe just a short summary of model side enumeration in this problem and its benefits